lib/engine/tile.rb

```diff
@@ -63,15 +63,20 @@ def self.part(type, params, cache)
       case type
       when 'path'
         params = params.map do |k, v|
-          case v[0]
-          when '_'
-            [k, cache[v[1..-1].to_i]]
+          case k
+          when 'terminal'
+            [k, v.to_i]
```

Comment:
no need to do a conversion here since any value w ill be truthy

Reply:
OK